openshift/manifests/20-deployment-openshift-operator-controller-operator-controller-controller-manager.yml

```diff
@@ -111,6 +117,14 @@ spec:
           secret:
             optional: false
             secretName: operator-controller-cert
+        - configMap:
+            name: operator-controller-trusted-ca-bundle
+            optional: false
+          name: trusted-ca-bundle
+        - configMap:
+            name: openshift-service-ca.crt
+            optional: false
+          name: service-ca
```

Comment:
For the metrics we are using the `/var/trusted-cas`
Just for I learn where we use this one? 
Why do we need pass both?
WDYT about we add a comment # Cert required for ... on top of both?